templates/vars-30.erb

```diff
@@ -0,0 +1,80 @@
+# easy-rsa parameter settings
```

Comment:
as per our [review guidelines](https://voxpupuli.org/docs/#reviewing-a-module-pr), new templates should be epp.

Reply:
No problem, have converted this

Reply:
I'm struggling to make the epp template work with the parameters from the defined type, `openvpn::ca`. Any guidance here? Should these be passed in as a parameter on the `epp()` function?

Reply:
Perhaps read this link will permit to understand the global context : 
https://tickets.puppetlabs.com/browse/PUP-7276

A possible way is like into `puppetlabs-apt` module with defined type `apt::setting`.
An other way, as you suggested, is to pass a hash as the last argument of the `epp()` function.